index.js

```diff
@@ -4,6 +4,22 @@
 module.exports = {
   name: 'ember-truncate',
   included: function(app) {
```

Comment:
The `app` parameter is no longer used, merely overwritten. Let's drop it from the function signature.